week3/homework/src/actions.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const { readFile, writeFile } = require('fs');
+const { promisify } = require('util');
+const uuid = require('uuid/v4');
+const read = promisify(readFile);
+const write = promisify(writeFile);
+const FILE_PATH = 'todo.json';
+const list = {};
+
+
+const fileContent = (todo) => {
+    return write(FILE_PATH, JSON.stringify(todo, null, 2));
+};
+
+const createTodo = async (todo) => {
+    if (!todo.description) throw new Error('Description is required');
+
+    const todoList = await readTheList();
+    const id = uuid();
+    list[id] = todo;
```

Comment:
You are adding each new to-do to a global variable and then pushing that object (eventually, with multiple to-dos) to the array. So this method doesn't work as intended.